examples/example-generation/types/typelist

```diff
@@ -0,0 +1,873 @@
+ObjectDetector
```

Comment:
That is actually true. Could you add this to the CMake stuff and update the patch?